### ISSUE TITLE
If the number of search results is large, they are not displayed on the screen.

### ISSUE THREAD

**Issue by misohouse** [LuatixHQ/opencti#3882]
**Current OpenCTI Version : 5.9.6 (latest)**

**Case : search google**
![Cyber threat intelligence platform 2023-07-31 13-45-54](https://github.com/OpenCTI-Platform/opencti/assets/19428462/3ca506ab-2b92-4e14-9965-739a8a613997)

Result
![11](https://github.com/OpenCTI-Platform/opencti/assets/19428462/743859f0-2148-426a-9cd4-35f6e9058bcb)

I thought the "https://google.com" didn't search.

But when I clicked "TYPE" column for sorting, I noticed the result was appeared.
![22](https://github.com/OpenCTI-Platform/opencti/assets/19428462/419f0c6d-6ead-488c-9ebb-d782a4bdd796)

When there are so many search results that it is difficult to show them all on one page, I suggest creating a number button at the bottom to move to the next page.

**Comment by misohouse** [comment]
(+) In the upper right corner, the entity number is also displayed as 0.
![Cyber threat intelligence platform 2023-07-31 15-01-19](https://github.com/OpenCTI-Platform/opencti/assets/19428462/8980177e-5a43-48e3-ab89-62a54647b7bd)

**Comment by misohouse** [comment]
I found the other thing.. when I search a word google.com for "https://www.google.com", there was no result.

It only appeared when I search a word www.google.

It's too hard to understand the mechanism of search engine.. 

![111](https://github.com/OpenCTI-Platform/opencti/assets/19428462/e43228ab-cb3e-406d-8a09-a353ead0ff90)
![222](https://github.com/OpenCTI-Platform/opencti/assets/19428462/ec114a14-fac4-46a6-aa8e-c16638af1ee3)
![333](https://github.com/OpenCTI-Platform/opencti/assets/19428462/3d090a61-2470-4448-95c4-e0c3cc7aa69d)
![444](https://github.com/OpenCTI-Platform/opencti/assets/19428462/240b6969-fe65-4de8-9799-8fce0e24da29)

**Comment by SouadHadjiat** [comment]
Hello @misohouse, 

Thank you for raising this issue, infinite loading for global search is indeed not working anymore along with the total count (only the first 25 results are displayed and scrolling doesn't display more), we'll fix it for the next release.

About the search engine, in the attack pattern example you provided, there is a google link in the description, this is why you see this result :

![Capture d'écran 2023-07-31 104736](https://github.com/OpenCTI-Platform/opencti/assets/668192/ebce107a-fda0-4c72-af70-e3096c7616c8)

As for the results of google.com search, I'm not sure to understand the issue, I just tested in the demo and there are results for google.com and https://www.google.com : https://demo.opencti.io/dashboard/search/https%3A%2F%2Fwww.google.com%2F

**Comment by misohouse** [comment]
Thank you for reply @SouadHadjiat

I'm trying to explain again because the demo link is not accessible.

I want to search Observations registered as www.google.com

So I typed google.com into the search box but didn't get any results.

But when I typed in www.google.com, the result came out.

I was embarrassed because I thought that searching only for google.com, the string included in the middle, should return results.

So I'm wondering if this is a bug, or if OpenCTI's search method is different from what I'm thinking.
